src/components/Admin/Trainer/Model.js

```diff
@@ -0,0 +1,231 @@
+import * as tf from '@tensorflow/tfjs';
+import {ControllerDataset} from './ControllerDataset';
+import getStore from '../../../utils/honestyStore.js';
+import * as MobileNet from '@tensorflow-models/mobilenet';
+
+class Model {
+	constructor(status) {
+		this.setStatus = status;
+		this.items = {unknown: {name: 'Unknown', id: 'unknown'}};
+
+		this.getName = this.getName.bind(this);
+		this.init = this.init.bind(this);
+		this.loadStore = this.loadStore.bind(this);
+		this.loadModel = this.loadModel.bind(this);
+		this.saveModel = this.saveModel.bind(this);
+		this.exportModel = this.exportModel.bind(this);
+		this.addExample = this.addExample.bind(this);
+		this.train = this.train.bind(this);
+		this.predict = this.predict.bind(this);
+
```

Comment:
Code would look cleaner if arrow functions and not bind

Reply:
These lines aren't necessary at all, arrow functions aren't needed. My bad.